assets/package.json

```diff
@@ -0,0 +1,139 @@
+{
+	"scripts": {
+		"build": "npm run build:production",
+		"prebuild:dev": "npm run remove-dist",
+		"build:dev": "webpack --mode=development --devtool cheap-module-eval-source-map --output-pathinfo",
+		"prebuild:production": "npm run remove-dist",
+		"build:production": "webpack --mode=production",
+		"build:test": "npm run build:production -- --include-tests",
+		"watch": "npm run build:dev -- --watch",
+		"remove-dist": "rimraf ../dist"
+	},
+	"dependencies": {
```

Comment:
@techanvil, all dependencies that have been moved to this `package.json` file, should be removed from the main `package.json` file if they are not used in other places of our codebase that doesn't have its own NPM workspace yet.

At the end, the main `package.json` file should only contain global level dependencies (like eslint, husky, etc). Everything else should be located in their workspaces.

Reply:
Thanks @eugene-manuilov, however we'll need to keep most of these in the main `package.json` until the [`storybook`](https://github.com/google/site-kit-wp/issues/10091) and [`tests/js`](https://github.com/google/site-kit-wp/issues/10094) workspaces have been added.

I'll another look through the list, though - I think there are a few which we can remove from the main `package.json`.

Reply:
Thanks, Tom.

Reply:
No problem - I've reviewed the dependencies and removed them from the main `package.json` where possible.